scripts/local.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+# This script is only meant to run from Makefile in the parent directory of this repository
+
+SCRIPTPATH="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+
+if [ $# -eq 0 ]
+  then
+    echo "No arguments supplied, this command requires a Github Action step tag to run, like:"
+    for job in $(egrep "\w+:$$" workflows/tests.yml | grep -v 'with:\|jobs:\|steps:' | cut -d ' ' -f 3 | cut -d ':' -f 1); do \
+	echo "- $job"
+    done
+    exit 1
+fi
+
+JOB=$1
+
+# ensure this is at the root
+cd "$SCRIPTPATH/.."
```

Comment:
nit / clarifying question: if this script is only meant to run in the root of the repo, would `$(pwd)` be alright instead?

Reply:
interesting, yes, I think that would work. I did it this way to ensure that regardless of where it runs it is always correct. Do you think that assurance is overkill?